CMakeLists.txt

```diff
@@ -71,5 +75,19 @@ install(DIRECTORY ${ASSETS_DIR} DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_N
 
 
 # CPack config
+set(CPACK_PACKAGE_NAME ${PROJECT_NAME})
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Chord naming application")
+set(CPACK_PACKAGE_VENDOR "Ters")
+set(CPACK_PACKAGE_VERSION_MAJOR ${PACKAGE_VERSION_MAJOR})
+set(CPACK_PACKAGE_VERSION_MINOR ${PACKAGE_VERSION_MINOR})
+set(CPACK_PACKAGE_VERSION_PATCH ${PACKAGE_VERSION_PATCH})
+set(CPACK_PACKAGE_ARCHITECTURE "amd64")
+set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "amd64")
```

Comment:
Do we need to set these values? I see cmake has some sensible defaults

Reply:
yeah, we just need to make sure build agent has dpkg installed to know the proper architecture